### PR TITLE
Homework_markup_1 fixed

### PR DESCRIPTION
**Pull request by niampire**

### CODE REVIEW

File: homeworks/kateryna.chepurna_niampire/homework_markup_1/index.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+    <link href="//fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800" rel="stylesheet" type="text/css">
+</head>
+<body>
+<header class="heading">
+    <div class="container">
+        <div class="heading-grid">
+            <a href="#">
+                <img class="logo" src="images/logo.png" title="dreams">
```
Comment (Nimmerz) [inline homeworks/kateryna.chepurna_niampire/homework_markup_1/index.html]: alt а не title

File: homeworks/kateryna.chepurna_niampire/homework_markup_1/index.html [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+            </a>
+            <nav class="nav">
+                <div class="nav-menu">
+                    <ul>
```
Comment (Nimmerz) [inline homeworks/kateryna.chepurna_niampire/homework_markup_1/index.html]: a - class must be nav-menu__list--link if you use BEM

File: homeworks/kateryna.chepurna_niampire/homework_markup_1/index.html (hunk 1 above)
Comment (niampire) [reply homeworks/kateryna.chepurna_niampire/homework_markup_1/index.html]: fixed

File: homeworks/kateryna.chepurna_niampire/homework_markup_1/index.html (hunk 2 above)
Comment (niampire) [reply homeworks/kateryna.chepurna_niampire/homework_markup_1/index.html]: fixed.
You may close this pull request because, homework_markup_1  was merged.

**Pull request closed without merging**